var/da/da_control/da_control.f90

```diff
@@ -690,4 +690,10 @@ module da_control
 
    logical, allocatable :: fgat_rain_flags(:)
 
+   integer, parameter :: no_thin           = 0
+   integer, parameter :: thin_single       = 1
+   integer, parameter :: thin_multi        = 2
+   integer, parameter :: thin_superob      = 3
+   integer, parameter :: thin_superob_hv   = 4
+
```

Comment:
Would be good if add some explanation about these variables. I do not see they are not used in other places of modified code. Do I miss something?

Reply:
They are used in the next coming PR.